ingest/annotations.py

```diff
@@ -102,52 +113,96 @@ def preprocess(self):
             log_exception(Annotations.dev_logger, Annotations.user_logger, msg)
             raise ValueError(msg)
 
-    def create_data_frame(self):
+    @staticmethod
+    def convert_header_to_multiIndex(df, header_names: List[Tuple]):
+        """Header in annotation files are represented as multi-index based on first 2 rows i.e causes
+            the first two rows in file to be headers.
+        Parameters
+        ----------
+            df (pandas dataframe) : Dataframe
+            columns (List[tuples]): Header names
+        Returns
+        -------
+            df (pandas dataframe): Dataframe with multi-indexed header
         """
-        - Create dataframe with proper dtypes to ensure:
-            - Labels are treated as strings (objects)
-            - Numeric annotations are treated as float32
-            - Numeric columns are rounded to 3 decimals points
+        index = pd.MultiIndex.from_tuples(header_names, names=["Name", "TYPE"])
+        df.columns = pd.MultiIndex.from_tuples(index)
```

Comment:
I separated this into a separate function because we've had issues with the way panda can parse headers. If the headers aren't read correctly into the data frame then we risk ingest failure or incorrect data association.